apps/src/music/blockly/blocks/events.js

```diff
@@ -1,14 +1,15 @@
+import musicI18n from '../../locale';
```

Comment:
ultra nit: slight preference for importing this near the end of the list, and keeping more module-specific and functional imports higher up.